code/modules/power/gourmonger.dm

```diff
@@ -12,16 +12,15 @@ var/global/gourmonger_saturation = 0
 	icon_living = "gourmonger"
 	icon_dead = "gourmonger"	//This shouldn't happen
 	faction = "gourmonger"
-	health = 125
-	maxHealth = 125
+	health = 60
+	maxHealth = 60
 	melee_damage_lower = 5
 	melee_damage_upper = 10	//The rads are the real danger
 	stat_attack = DEAD	//So it attacks corpses.
 	search_objects = 1	//Searches objects but doesn't ignore people. The ignoring before hanger is in its CanAttack()
-	environment_smash_flags = SMASH_CONTAINERS
 	wanted_objects = list(/obj/item/weapon/reagent_containers/food/snacks)
 	meat_amount = 1
-	speed = 1.2	//Able to just barely outrun them
+	speed = 2	//Able to just barely outrun them -- How are these fat fucks able to move so fast?
```

Comment:
They aren't fat fucks, that's the point, they're literally a stomach with legs that instantly turns food into rads.

Reply:
Sounds fat to me

Reply:
you can't be fat if you don't produce fat